python/servo/try_parser.py

```diff
@@ -89,26 +91,44 @@ def handle_preset(s: str) -> Optional[JobConfig]:
 
     if s == "linux":
         return JobConfig("Linux", Workflow.LINUX, unit_tests=True)
-    elif s in ["perf", "linux-perf", "bencher"]:
-        return JobConfig("Linux perf", Workflow.LINUX, bencher=True)
+    elif s in ["linux-bencher"]:
+        return JobConfig("Linux (Bencher)", Workflow.LINUX, bencher=True)
+    elif s == "linux-production-bencher":
+        return JobConfig("Linux (Production, Bencher)", Workflow.LINUX, bencher=True, profile="production")
```

Comment:
Why do we need to write `(Production, Bencher)` now that we could just reuse modifier code?

Reply:
Actually it's just semi-handwritten atm because I can't figure out some complications earlier, but it suddenly struck me that I'm just overthinking some issue, I'll rework the modifier workflow.